panel/widgets/indicators.py

```diff
@@ -274,7 +268,7 @@ class Number(ValueIndicator):
     title_size = param.String(default='18pt', doc="""
         The size of the title given by the name.""")
 
-    _rename: ClassVar[Mapping[str, str | None]] = {}
+    _rename: ClassVar[Mapping[str, str | None]] = {'name': 'name'}
```

Comment:
I'm not sure I follow this rename... Is it because the parent class rename `name`?